docs/interfaces.md

```diff
@@ -176,6 +176,12 @@ playback.
 
 * Auto-Connect: yes
 
+### pluggable-storage
+
+Can access files from pluggable storage in /media and /run/media
```

Comment:
Thanks for these latest changes!

Please adjust this to be 'removable-media', move so it is alphabetical and adjust the text to be 'removable storage'.